script.js

```diff
@@ -1,12 +1,169 @@
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
+  rootElem.textContent = "";
+
+  //top container which will wrap all movieCart
+  const topContainer = document.createElement("div");
+  topContainer.classList.add("top-container");
+  rootElem.appendChild(topContainer);
+  for (let i = 0; i < episodeList.length; i++) {
+    //this is movieCard in which all title, photo and summery will be appended.
+    const movieCard = document.createElement("div");
+    movieCard.classList.add("movie-card");
+    const testElement = document.createElement("p");
+    movieCard.appendChild(testElement);
+    topContainer.appendChild(movieCard);
+
+    //Assigning TV episodes in p element
+    let episodeName = document.createElement("p");
+    episodeName.classList.add("epName");
+    movieCard.appendChild(episodeName);
 
+    //assigning the season number & episode number//
+    episodeSeason = episodeList[i].season;
+    episodeNumber = episodeList[i].number;
+    if (episodeSeason < 10) {
+      episodeSeason = "0" + episodeSeason;
+    }
+    if (episodeNumber < 10) {
+      episodeNumber = "0" + episodeNumber;
+    }
+    episodeName.innerText =
+      episodeList[i].name +
+      " " +
+      "-" +
+      " " +
+      "S" +
+      episodeSeason +
+      "E" +
+      episodeNumber;
+
+    //creating image container div//
+    let imageContainer = document.createElement("div");
+    imageContainer.classList.add("image-container");
+    movieCard.appendChild(imageContainer);
+
+    //creating episodeImage
+    let episodeImage = document.createElement("img");
+    imageContainer.appendChild(episodeImage);
+    episodeImage.src = episodeList[i].image.medium;
+
+    //creating summary div//
+    let summaryContainer = document.createElement("div");
+    summaryContainer.classList.add("summary-container");
+    movieCard.appendChild(summaryContainer);
+
+    //creating episodeSummary//
+    const episodeSummary = document.createElement("p");
+    summaryContainer.appendChild(episodeSummary);
+    summaryContainer.innerHTML = `${episodeList[i].summary}`;
+
+    ///creating div for level 300 and appended rootElm inside
+    let episodeDivContainer = document.createElement("div");
+    episodeDivContainer.classList.add("episode-container");
+    episodeDivContainer.id = "episode-container" + i;
+    rootElem.appendChild(episodeDivContainer);
+  }
+
+  //creating select bar for episodes for level 300
+
+  let selectElem = document.querySelector("#selector");
+  let optionElem = document.createElement("option");
+  optionElem.innerText = "Please Choose Your Episodes";
+  selectElem.appendChild(optionElem);
+
+  allEpisodes.forEach((ele) => {
+    let options = document.createElement("option");
+    options.value = ele.name;
+    options.innerText = `${ele.name} - S${ele.season
+      .toString()
+      .padStart(2, "0")}E${ele.number.toString().padStart(2, "0")}`;
```

Comment:
It would be more readable to extract number formatting into it's own function and reuse it for season and episode

Reply:
But good use of `.padStart()` and template literals